latest/PIT/pit_common.yaml

```
@@ -0,0 +1,188 @@
+          File creation date-time in ISO 8601 format with UTC time
+          scale.
+        tag: tag:stsci.edu:asdf/time/time-1.*
+      start_time:
+        title: Start time
+        description: |
+          Start time of data acquisition in ISO 8601 format.
+          tag: tag:stsci.edu:asdf/time/time-1.*
+      end_time:
+        title: End time
+        description: |
+          End time of data acquisition in ISO 8601 format.
+        tag: tag:stsci.edu:asdf/time/time-1.*
+      exposure_time:
+        # Optional
+        title: Exposure time (s)
+        description: |
+          The amount of time elapsed between the start and the end of
+          the observation or data stream.
+        type: number
+        unit: "s"
+      effective_exposure_time:
+        title: Effective exposure time (s)
+        description: |
+          The total accumulated effective exposure duration for this
+          data product, excluding dead time and lost time.
+        type: number
+        unit: "s"
+      instrument:
+        title: Instrument observing configuration
+        type: object
+        properties:
+          name:
+            title: Instrument name
+            anyOf:
+              - type: string
+              - type: "null"
+          detector:
+            title: Detector
+            anyOf:
+              - type: string
+              - type: "null"
+          optical_element:
+            title: Optical element
+            description: |
+              Name of filter, disperser, or other optical element.
+            anyOf:
+              - type: string
+              - type: "null"
+        required: [name, detector, optical_element]
+      telescope:
+        title: Telescope name
+        description: |
+          Name of telescope, typically Roman.
+        anyOf:
+          - type: string
+          - type: "null"
+      target_coordinates:
+        title: Position and region for this product
+        type: object
+        properties:
+          reference_frame:
+            anyOf:
+              - type: string
+              - type: "null"
+          ra:
+            title: Right ascension (deg)
+            description: |
+              Characteristic right ascension in degrees; typically the
+              coordinate of the source of interest, or the center of
+              the spatial image. May be null for large survey images,
+              simulated data, etc.
+            anyOf:
+              - type: number
+              - type: "null"
+            unit: "deg"
+          dec:
+            title: Declination (deg)
+            description: |
+              Characteristic declination in degrees; typically the
+              coordinate of the source of interest, or the center of
+              the spatial image. May be null for large survey images,
+              simulated data, etc.
+            anyOf:
+              - type: number
+              - type: "null"
+            unit: "deg"
+          s_region:
```

```suggestion
          s_region:
```
Would you expand on why this is under `target_coordinates`?

---

I'm happy to move it out to the top level, I think. (My thinking had been: it's the target sky footprint, which is basically target coordinates. Mostly I wanted it in a leaf with the reference_frame, but I forgot that STC-S s_region strings specify the reference frame anyway, so I guess that's not necessary.)

---

Thanks. This is helpful clarification. For a repackaged L3 product would this be the same value as `meta.wcsinfo.s_region`?

---

Yes. (Which is also why it's in pit_common instead of pit_basic.) In the absence of the other wcsinfo metadata though, it's not really "info about the WCS", IMO, so I'd mildly prefer not to put it under a wcsinfo key for custom PIT products.

---

But if you think keeping mimicking the key structure is worth the terminology confusion, I'd be fine with a wcsinfo key holding this, pixel_scale, and pixel_shape.

---

Thanks. I should clarify that I'm not at the moment advocating for a `wcsinfo` key, more just checking that I'm understanding that:
- `target_coordinates.s_region` is the image footprint
- `target_coordinates.ra` might be the image center ra or the source of interest ra (same for ...dec)
- `spatial_dimensions` is the image size in pixels

---

Your understanding is correct!